Jacky's Pi Sys/pi-systems_door-subsystem.py

```diff
@@ -1,39 +1,58 @@
 import importlib
+from enum import IntEnum
 subsys = importlib.import_module('pi-systems_subsystem-base')
 comms = importlib.import_module('pi-systems_communications')
-from enum import IntEnum
+
 
 class DoorSubsystem(subsys.Subsystem):
 
     class Procedure(IntEnum):
-        OpenDoor = ord("o")
-        CloseDoor = ord("c")
-
+        OpenDoor = 111
```

Comment:
OpenDoor and CloseDoor are not procedures - they are present in the setDoorState and getDoorState messages. Suggest moving OpenDoor and CloseDoor to a different enum, i.e. DoorRequest, DoorState...